benches/coprocessor_executors/mod.rs

```diff
@@ -1,16 +1,4 @@
-// Copyright 2018 PingCAP, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
+// Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
This is nit-picky and I apologise in advance, but could we keep the blank line between the comment and the first line of code? Makes things prettier IMO.

Reply:
🤣 https://github.com/tikv/tikv/pull/4438/commits/80c83dcc70fd3414db8c8b2acc15849bfcc2e1fe